app/src/main/java/de/hhn/aib/labsw/blackmirror/dataclasses/Widget.kt

```diff
@@ -1,7 +1,25 @@
 package de.hhn.aib.labsw.blackmirror.dataclasses
 
-class Widget(name: String, x: Int, y: Int) {
+/**
+ * This class represents the widgets on the mirror.
+ *
+ * @author Selim Özdemir, Niklas Binder
+ * @version 24-05-2022
+ */
+class Widget(type: WidgetType, x: Int, y: Int) {
     private var xVal: Int = x
     private var yVal: Int = y
```

Comment:
theres actually no need to have different names but this doesn't even matter because you'd move these attributes into the primary constructor anyway